app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/item/VaultItemViewModel.kt

```diff
@@ -194,8 +195,8 @@ class VaultItemViewModel @Inject constructor(
             is VaultItemAction.Common.ConfirmCloneWithoutFido2CredentialClick -> {
                 handleConfirmCloneClick()
             }
-
```

Comment:
This formatting was correct previously, make sure you have the Bitwarden formatter [setup](https://github.com/bitwarden/android?tab=readme-ov-file#setup).